src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public abstract class Manager extends AbstractEmployee {
+
+    public ArrayList<AbstractEmployee> listOfEmployees;
```

Comment:
- Powinien być private oraz powinno się wykorzystawać jak najwyższy poziom abstrakcji jaki satysfakcjonuje nasze wymagania, tutaj wykorzystując interfejs List w taki sposób : `private List<AbstractEmployee> listOfEmployees;`
Chcąc później zmienić wykorzystywaną implementację wystarczyłoby to zmienić jedynie w konstruktorze np.
z `listOfEmployees = new ArrayList<>();`
na `listOfEmployees = new LinkedList<>()';`
trzymając konkretną implementację w polu musielibyśmy dodatkowo zmieniać w całej klasie typy np. w getterze na liste mielibyśmy ArrayList<> getListOfEmployees(); musielibyśmy zamienić ArrayList na LinkedList, tak samo we wszystkich parametrach itp. gdybyśmy zwracali interfejs List nic nie trzeba by było zmieniać.